satellite/contact/service.go

```diff
@@ -17,24 +22,78 @@ var Error = errs.Class("contact")
 
 var mon = monkit.Package()
 
+// Config contains configurable values for contact service
+type Config struct {
+	ExternalAddress string `user:"true" help:"the public address of the node, useful for nodes behind NAT" default:""`
+}
+
+// Conn represents a connection
+type Conn struct {
+	conn   *grpc.ClientConn
+	client pb.NodesClient
+}
```

Comment:
there is already client code at `satellite/contact/client.go`, would be nice to merged these 2

Reply:
I'll take a look

Reply:
So the grpc.ClientConn in this struct is created in FetchInfo/dialNode via transport.DialNode but the one in the client is created by transport.DialAddress in newClient. I think they're different enough that it would complicate things for now if we combined them.

Reply:
sounds good